coffee_core/src/coffee.rs

```diff
@@ -288,50 +297,71 @@ impl PluginManager for CoffeeManager {
         self.storage.store(&self.storage_info()).await
     }
 
+    async fn remote_sync(&mut self) -> Result<(), CoffeeError> {
```

Comment:
```suggestion
    async fn remote_sync(&mut self) -> Result<(), CoffeeError> {
```

can you call this command `nurse`?

Reply:
The command is called nurse. This is just a function to check for the local folder that we call with every other command.